pkg/detector/ospkg/rootio/rootio.go

```diff
@@ -40,6 +40,10 @@ func NewScanner(baseOS ftypes.OSType) *Scanner {
 		comparer = version.NewDEBComparer()
 		vsg = rootio.NewVulnSrcGetter(vulnerability.Ubuntu)
 		versionTrimmer = version.Minor
+	case ftypes.Rocky:
+		comparer = version.NewDEBComparer()
```

Comment:
We should add RPM comparer:
https://github.com/aquasecurity/trivy/blob/c80310d7690d8aeb7d3d77416c18c0c8b9aebe17/pkg/detector/ospkg/rocky/rocky.go#L62